### ISSUE TITLE
Unexpected behavior in reaction_rules

### ISSUE THREAD

**Issue by formersbach** [biomass-dev/biomass#187]
### Description:

Equation where 1 reactant reacts to 2 products is not covered by `state_transition` function but rather raises `ValueError` in `_bind_and_dissociate`.
I was trying to create a transcription event with mass action kinetics. I'm not sure whether this is "right" or "wrong", maybe you have an idea @himoto !

### Reproduce:

Text file containing:
```
nNfk --> nNfk + mIkb
```

Convert to model:
```
from biomass import Text2Model, create_model
model = Text2Model("test.txt")
model.convert(overwrite=True)
```
### Observed behavior

Raises : 
```
ValueError
line1: nNfk <- Use a different name.
```
### Desired behavior

ODE:
```
dmIkb/dt = kf1 * nNfk
dnNfk/dt = 0
```

**Comment by himoto** [comment]
Hi @formersbach, thanks for your report.
```
nNfk --> nNfk + mIkb
```
If I understand your comment correctly, the rule above should be equal to:
```
nNfk synthesizes mIkb
```
to achieve the desired behavior.

I sent a PR to address this issue in progress #188.
I would be grateful if we can discuss this update, thanks!

**Comment by himoto** [comment]
This issue is resolved in #188 and I would like to close this.
Please raise a new issue if you have any questions and suggestions.
Thank you.
